pkgs/development/libraries/gtk+/3.x.nix

```diff
@@ -21,12 +21,14 @@ stdenv.mkDerivation rec {
     sha256 = "1l45nd7ln2pnrf99vdki3l7an5wrzkbak11hnnj1w6r3fkm4xmv1";
   };
 
+  NIX_LDFLAGS = stdenv.lib.optionalString stdenv.isDarwin "-lintl";
+
   nativeBuildInputs = [ pkgconfig gettext gobjectIntrospection perl ];
 
   buildInputs = [ libxkbcommon ];
   propagatedBuildInputs = with xlibs; with stdenv.lib;
-    [ expat glib cairo pango gdk_pixbuf atk at_spi2_atk ]
-    ++ optionals stdenv.isLinux [ libXrandr libXrender libXcomposite libXi libXcursor wayland ]
+    [ expat glib cairo pango gdk_pixbuf atk at_spi2_atk libXi libXfixes libXrandr libXrender libXcomposite libXcursor ]
```

Comment:
Adding `libXfixes` (and changing the order of the build inputs) will cause rebuilds on linux, and many of them given that this is gtk :)

Can you revert to the original order and add `libXfixes` as an optional input for darwin? (Unless there's a good reason for linux to have it too, idk what it actually provides..)